ppr-ui/src/components/common/CertifyInformation.vue

```diff
@@ -109,7 +113,7 @@ export default defineComponent({
   props: {
     sectionNumber: {
       type: Number,
-      default: 2
+      required: false
```

Comment:
@orelbn please mention in the ticket to do light regression if this component is used elsewhere beyond unit notes.

Reply:
I did some light regression testing myself, but will also mention in the ticket.